src/server.rs

```diff
@@ -341,6 +343,12 @@ impl Inner {
     }
 }
 
+impl Debug for Inner {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        write!(f, "Server {:?}", self.bind_addrs)
```

Comment:
I think `Formatter::debug_struct`[0] is better than handwritten.

[0] https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct

Reply:
I think it's fine, since only one field is used.